communication/src/main/java/org/jboss/da/communication/pnc/impl/PNCConnectorImpl.java

```diff
@@ -11,38 +11,48 @@
 import org.jboss.da.communication.pnc.model.Project;
 import org.jboss.resteasy.client.ClientRequest;
 import org.jboss.resteasy.client.ClientResponse;
+import org.jboss.resteasy.util.GenericType;
 
+import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 import javax.ws.rs.core.MediaType;
 
-import java.util.Arrays;
 import java.util.List;
 
-import javax.enterprise.context.ApplicationScoped;
-
 @ApplicationScoped
 public class PNCConnectorImpl implements PNCConnector {
 
     @Inject
-    private Configuration config;
+    private PNCAuthentication pncAuthenticate;
+
+    private String pncBaseUrl;
+
+    @Deprecated
+    public PNCConnectorImpl() {
+    }
```

Comment:
It is there because of the Constructor injection. CDI is complaining when you use constructor injection and you have no constructor without parameters